stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
```

Comment:
So we should have asyncUnimplementedUnaryCall and asyncUnimplementedStreamingCall? The only difference seems to be whether we need to return a NOOP StreamObserver.